.github/workflows/release.yaml

```diff
@@ -29,14 +29,27 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Release Maven package
+
+      - name: Release Pulsar Flink with Scala 2.11
```

Comment:
I think its better to use [maxtrix](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) to summarize `scala-2.11` and `scala-2.12`. This should reduce the amount of code and increase the performance.

Reply:
OK, I'll fix this.

Reply:
I think that I have fixed this. Can you help me review it? @timmyyuan